feat: add readme for logging operator add-on (#18)
mia-platform/distribution: addons/logging-operator/logging-stack/README.md

# Logging system for Logging operator 

> This add-on define and deploys the logging system of the Logging operator. It consists of the necessary `FluentD` and `FluentBit` configurations. 

Reviewer: ```suggestion
This add-on defines and deploys the logging stack of the Logging operator. It consists of the necessary `FluentD` and `FluentBit` configurations. 
```

> 

Reviewer: ```suggestion
To work properly it needs the Logging Operator module.

```

## What the Add-On Contains

**Resurces:**

> - **Logging:** Logging operator's CRD that define configurations fot FluentD and FluentBit.
> - **Networkpolicies:** Networkpolicy neede to allow traffic for FluentBit.

Reviewer: ```suggestion
- **Logging:** Logging CRD that defines configurations for FluentD and FluentBit.
- **Network policies:** Network policy needed to allow traffic for FluentBit.
```

## Compatibility Matrix

> | Add-On Version | Cert Manager Module Version | kube-green Module Version |
> |----------------|-----------------------------|---------------------------|
> | 1.0.0          | 1.24.0                      | 1.24.0                    |

Reviewer: ```suggestion
| Add-On Version | Logging Operator Module Version |
|----------------|---------------------------------|
| 1.0.0          | 1.24.0                          |
```

## User customization

> No customization is needed

Reviewer: ```suggestion
No customization is needed. The add-on is ready-to-use.
```
